.github/workflows/commit.yml

```diff
@@ -12,3 +12,5 @@ jobs:
       - name: check-for-cc
         id: check-for-cc
         uses: agenthunt/conventional-commit-checker-action@v1.0.0
+        with:
+          pr-title-regex: "^(.+)(?:(([^)s]+)))?!?: (.+)"
```

Comment:
what's this changing from the defaults?

Reply:
it adds the ability to add the `!` as I described in the `CONTRIBUTING.md` to mark breaking changes